### ISSUE TITLE
leap: update tests to version 1.1.0

### ISSUE THREAD

**Issue by N-Parsons** [exercism/python#1017]
As identified in #784, some of the exercise test-suites are out-of-date, `leap` is one of them. It would therefore be really helpful for people to go through and update the test suite to match the canonical data.

Tests should typically be ordered according to their order in the canonical data, since this makes it easier to check and update them in future. If you think that there's a better order, please let us know your reasoning in your PR.

If there are some additional tests that currently exist in this track but which aren't in the canonical data, these can be kept if you think that they are useful. They should be placed at the bottom of the test cases, and it would also be super-helpful if you could add a comment before them saying `# Additional tests for this track` to make it clear that they aren't part of the canonical data.

The version string should be placed just before the first test class and followed by a blank line. For this exercise, the version string is:
```
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
```

**Test suite:** [exercism/python/exercises/leap/leap_test.py](https://github.com/exercism/python/blob/master/exercises/leap/leap_test.py)
**Canonical data:** [exercism/problem-specifications/exercises/leap/canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/leap/canonical-data.json)

**Comment by kishan3** [comment]
@N-Parsons I think this file is already updated.

**Comment by N-Parsons** [comment]
Thanks for pointing this out. Looks like it got resolved in #979 when there was another update made.
